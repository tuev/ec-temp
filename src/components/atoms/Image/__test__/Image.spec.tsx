import React from 'react'
import { shallow } from 'enzyme'
import { Image } from '..'
import toJson from 'enzyme-to-json'

describe('Image', () => {
  it('Image no width and height attribute', () => {
    const AppImage = shallow(<Image src="./img.jpg" alt="shirt" />)

    expect(toJson(AppImage)).toMatchSnapshot()
  })
})
