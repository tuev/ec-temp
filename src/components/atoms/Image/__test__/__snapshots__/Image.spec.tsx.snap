// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Image no width and height attribute 1`] = `
<styled.img
  alt="shirt"
  src="./img.jpg"
/>
`;
